shipment_advice/models/shipment_advice.py

```diff
@@ -414,8 +410,16 @@ def button_open_deliveries_in_progress(self):
                 domain.append(("move_lines.shipment_advice_id", "=", False))
             if self.carrier_ids:
                 domain.append(("carrier_id", "in", self.carrier_ids.ids))
-            pickings = self.env["stock.picking"].search(domain)
-            action["domain"] = [("id", "in", pickings.ids)]
+        return domain
+
+    def button_open_deliveries_in_progress(self):
+        action = self.env.ref("stock.action_picking_tree_all").read()[0]
```

Comment:
future proof w/ `self.env['ir.actions.act_window'].for_xml_id('stock', 'action_picking_tree_all')`?

Reply:
I'll do another PR once this one is fixed & merged

Reply:
Ported here: https://github.com/OCA/stock-logistics-transport/pull/77